### ISSUE TITLE
Gnome dash persists and desktop overview at loging 

### ISSUE THREAD

**Issue by BenjiSan** [micheleg/dash-to-dock#1563]
Hello. When activating "dash to dock" the gnome dash still appears in the lower part of the screen (in the applications view and in "overview").
Also, when you log in, the desktop is shown the way gnome would (overview).
When you reactivate the ubuntu dock the problem disappears.
I have installed dash to dock from https://extensions.gnome.org/
I am using ubuntu 21.10

![Captura de pantalla de 2021-10-25 15-35-28](https://user-images.githubusercontent.com/13210014/138770075-64107f73-6a1f-428b-8bc3-2364de2e4d0d.png)
![Captura de pantalla de 2021-10-25 15-21-29](https://user-images.githubusercontent.com/13210014/138770102-158ac7ac-7e08-406f-b18c-2b168bb06486.png)

**Comment by 3v1n0** [comment]
Do you have other extensions enabled?

**Comment by BenjiSan** [comment]
> Do you have other extensions enabled?

Yes, I have several extensions installed and enabled. However, the problem continues even if I have disabled all of them. Even on a clean install of Ubuntu it happens (the two problems - double dock and desktop overview at start). In Fedora 35 (VM clean installation) only the desktop overview at start occurs.
![Captura de pantalla de 2021-11-17 03-11-21](https://user-images.githubusercontent.com/13210014/142172134-05c8ba07-71e5-444d-bf37-57127548a761.png)
![Captura de pantalla de 2021-11-17 03-19-07](https://user-images.githubusercontent.com/13210014/142172468-3961450f-60d9-4f7d-ac9e-d001145af380.png)
